pkg/image/export/main.go

```diff
@@ -262,3 +278,110 @@ func findStrings(obj map[string]interface{}, found map[string]bool) {
 		}
 	}
 }
+
+func getImagesFromCharts(path string) ([]string, error) {
+	var images []string
+	imageMap := map[string]struct{}{}
+	chartVersion, err := getChartAndVersion(path)
+	if err != nil {
+		return nil, err
+	}
+	if err := filepath.Walk(path, func(p string, info os.FileInfo, err error) error {
+		return walkFunc(imageMap, chartVersion, path, p, info, err)
+	}); err != nil {
+		return images, err
+	}
+	for value := range imageMap {
+		images = append(images, value)
+	}
+	return images, nil
+}
+
+func getChartAndVersion(path string) (map[string]string, error) {
+	rtn := map[string]string{}
+	helm := libhelm.Helm{
+		LocalPath: path,
+		IconPath:  path,
+		Hash:      "",
+	}
+	index, err := helm.LoadIndex()
+	if err != nil {
+		return nil, err
+	}
+	for k, versions := range index.IndexFile.Entries {
+		var maxVersion *libhelm.ChartVersion
+		for _, version := range versions {
+			if maxVersion == nil || version != nil && versionutils.GreaterThan(version.Version, maxVersion.Version) {
+				maxVersion = version
+			}
+		}
+		if maxVersion != nil {
+			rtn[k] = maxVersion.Dir
+		}
+	}
+
+	return rtn, nil
+}
+
+func walkFunc(images map[string]struct{}, versions map[string]string, basePath, path string, info os.FileInfo, err error) error {
+	relPath, err := filepath.Rel(basePath, path)
+	if err != nil {
+		return err
+	}
+	found := false
+	for _, v := range versions {
+		if strings.HasPrefix(relPath, v) {
+			found = true
+			break
+		}
+	}
+	if !found || info.Name() != "values.yaml" {
+		return nil
+	}
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return err
+	}
+	dataInterface := map[interface{}]interface{}{}
+	if err := yaml.Unmarshal(data, &dataInterface); err != nil {
+		return err
+	}
+
+	if err := walkthroughMap(dataInterface, func(inputMap map[interface{}]interface{}) error {
+		return generateImages(inputMap, images)
+	}); err != nil {
```

Comment:
I saw there aren't any errors will be returned in `generateImages`, should we make `walkthroughMap` return none error?